addons/hr_timesheet/__init__.py

```diff
@@ -32,3 +32,14 @@ def create_internal_project(cr, registry):
     } for task in project_ids.task_ids.filtered(lambda t: t.company_id in admin.employee_ids.company_id)])
 
     _check_exists_collaborators_for_project_sharing(env)
+
+def uninstall_hook(cr, registry):
```

Comment:
Can you rename this method into `_uninstall_hook` ? 🙂

Reply:
@xavierbol , Done!